test/signalrclient-e2e-tests/hub_connection_tests.cpp

```diff
@@ -0,0 +1,191 @@
+// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+#pragma once
+
+#include "stdafx.h"
+#include <string>
+#include "cpprest\details\basic_types.h"
+#include "cpprest\json.h" 
+#include "connection.h"
+#include "hub_connection.h"
+#include "console_log_writer.h"
+
+using namespace signalr;
+using namespace utility;
+using namespace std;
+
+TEST(hub_connection_tests, send_message)
+{
+	shared_ptr<log_writer> Log_writer(make_shared<console_log_writer>());
+
+	hub_connection hubConn(L"http://localhost:8081/SignalR", L"", trace_level::all, Log_writer);
```

Comment:
`L` is bad. Either use `_XPLASTR` or `U` - I am thinking about moving the entire project from `_XPLATSTR` to `U`